syntax/status.sublime-syntax

```diff
@@ -39,6 +39,17 @@ contexts:
         - match: '^  [ \-] (.+)\n$'
           captures:
               1: meta.git-savvy.status.file gitsavvy.gotosymbol
+    - match: ^  ((LOCAL)|(REMOTE \(origin\))):$
+      comment: branch-names
+      captures:
+        0: keyword.other.git-savvy.branch-header
+      push:
+        - meta_scope: meta.git-savvy.branch.section
+        - match: ^$
+          pop: true
+        - match: '^  . [A-Za-z0-9]{7} (.+)\n$'
+          captures:
+              1: meta.git-savvy.branches.branch gitsavvy.gotosymbol
```

Comment:
I don't understand why you updated `syntax/status.sublime-syntax`. Would you explain why you did it? Or show one example of what is fixes?